tests/e2e/test_pvc_creation_performance.py

```diff
@@ -165,3 +167,104 @@ def test_multiple_pvc_creation_after_deletion_performance(
         logging.info(
             f"{number_of_pvcs} PVCs creation time took less than a 45 seconds"
         )
+
+    def test_pvc_reattach_time_performance(self, pvc_factory, teardown_factory):
+        """
+        Test assign nodeName to a pod using RWX pvc
+        Performance in test_multiple_pvc_creation_measurement_performance
+        Each kernel (unzipped) is 892M and 61694 files
+        """
+        interface = constants.CEPHFILESYSTEM
+        kernel_url = 'https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.19.5.tar.gz'
+        download_path = 'tmp'
+        # Number of times we copy the kernel
+        copies = 3
+
+        # Download a linux Kernel
+        import os
+        dir_path = os.path.join(os.getcwd(), download_path)
+        file_path = os.path.join(dir_path, 'file.gz')
+        if not os.path.exists(dir_path):
+            os.makedirs(dir_path)
+        urllib.request.urlretrieve(kernel_url, file_path)
+
+        worker_nodes_list = helpers.get_worker_nodes()
+        assert (len(worker_nodes_list) > 1)
+        node_one = worker_nodes_list[0]
+        node_two = worker_nodes_list[1]
+
+        # Create a RWX PVC
+        pvc_obj = pvc_factory(
+            interface=interface, access_mode=constants.ACCESS_MODE_RWX,
```

Comment:
why not use `constants.CEPHFILESYSTEM` directly?

Reply:
ok